formver.compiler-plugin/core/src/org/jetbrains/kotlin/formver/core/linearization/LinearizationContext.kt

```diff
@@ -44,6 +46,8 @@ interface LinearizationContext {
     fun addDeclaration(decl: Declaration)
 
     fun addModifier(mod: StmtModifier)
+
+    fun emitAssignment(lhs: ExpEmbedding, rhs: ExpEmbedding): Exp?
```

Comment:
To conform to the general pattern, please use `add` instead of `emit.

Also, what is the interpretation of the returned `Exp`?